### PR TITLE
Quick UI + Profile Page updates

### PR DESCRIPTION
**Pull request by brianjcarroll**
This PR does the following:

* Pluralizes or Singularizes the word "Hunter" when appropriate
* Adds a "Copy" button to quickly copy a user npub (and copies it in npub encoding)
* Wraps github and website data in links to the corresponding link
* Adds the osjf npub as a vanity url (osjf@fundsolvr.com)

### CODE REVIEW

File: public/.well-known/nostr.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "names": {
-    "chris": "220522c2c32b3bf29006b275e224b285d64bb19f79bda906991bcb3861e18cb4"
+    "chris": "220522c2c32b3bf29006b275e224b285d64bb19f79bda906991bcb3861e18cb4",
+    "osjf": "1eef661d4e7dafc7f195ca328b0d7694fe8436e39c1e0336f80bc7e707f74d97"
```
Comment (tvolk131) [inline public/.well-known/nostr.json:4]: I'm not sure what this is for - @ChristianChiarulli can you verify that this looks right?

File: src/components/profile/ProfileCard.tsx [hunk 2]
```diff
@@ -56,6 +56,14 @@ function trimNip05(nip05Id: string | undefined) {
   return nip05Id;
 }
 
+async function copyText(text: string) {
+  try {
+    await navigator.clipboard.writeText(text);
+  } catch (err) {
+    console.error("Failed to copy: ", err);
```
Comment (tvolk131) [inline src/components/profile/ProfileCard.tsx:63]: Should we leave a TODO comment to eventually surface this error to the UI as a toast?

**Review by tvolk131 — CHANGES_REQUESTED** [review]
Just one small suggestion, otherwise LGTM!

**Review by tvolk131 — APPROVED** [review]

File: public/.well-known/nostr.json (hunk 1 above)
Comment (ChristianChiarulli) [reply public/.well-known/nostr.json:4]: this is for nip05

File: src/components/profile/ProfileCard.tsx (hunk 2 above)
Comment (ChristianChiarulli) [reply src/components/profile/ProfileCard.tsx]: we can probably just put in an issue

Comment (ChristianChiarulli) [reply src/components/profile/ProfileCard.tsx]: nvm it's handled

**Review by ChristianChiarulli — APPROVED** [review]

**Pull request merged**